core/src/main/java/org/pentaho/platform/util/ActionUtil.java

```diff
@@ -94,6 +94,9 @@
     KEY_MAP.put( ActionUtil.QUARTZ_RESTART_FLAG, ActionUtil.INVOKER_RESTART_FLAG );
   }
 
+  public static final String WORK_ITEM_XMS = "JVM_XMS";
+  public static final String WORK_ITEM_XMX = "JVM_XMX";
+
```

Comment:
I see why the declaration of these variables would be placed in pentaho-platform's core code ( to conform to the location where all the other variables are at as well ).

However, I see the other variables as being applicable for any and all `IAction`, while these JVM ones seem to be applicable only to someone who would care to have a notion of jvm settings running somewhere , such as a docker container.

So, by this line of reasoning, I'd have these variables declared in the only place they make sense, which is in `MesosChronosActionInvoker`

Reply:
It makes sense, I will close this PR and include these variables directly in the `MesosChronosActionInvoker` instead